CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
+
+## About the Project
+
+Scaffold-Eth 2 is a starter toolkit to build applications on Ethereum. It's a minimal and forkable repo providing builders a MVP to create decentralized applications.
+
+Read the [README](README.md) to get an overview of the project. 
+
+## Project Status
+
+The project is under active development. You can view the open Issues, follow along the development process and contribute to the project. 
+
+## Getting started
+
+You can contribute to this repo in many ways:
+- Solve open issues 
+- Report bugs or feature requests
+- Improve the documentation 
+
+Contributions are made to via Issues and Pull Requests (PRs). A few general guidelines for contributions:
+
+- Search for existing Issues and PRs before creating your own.
+- Contributions should only fix/add the functionality in the issue OR address style issues, not both.
+- If you’re running into an error please give context and explain what you’re trying to do and how to reproduce the error. 
+- Please use the same formatting in the code repository. 
+- Include the test files if you are modifying the code. 
+- If applicable, please edit the README.md file to reflect the changes. 
+
+### Issues
+
+Issues should be used to report problems, request a new feature, or to discuss potential changes before a PR is created.
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/scaffold-eth/se-2/issues) to find one that interests you. 
+
+If there is a contributor working on the issue they will be assigned to the individual. If you find an issue to work on, you are welcome to open a PR with a fix for it.
+
+#### Create a new issue
+
+If a related issue doesn't exist, you can open a new issue. 
```

Comment:
I'll add some tips here like:

- Over-communicate / As much context as possible
- Steps to reproduce / Reason to add the feature.
- Screenshots
- Videos, etc.
- Context